### ISSUE TITLE
Is it possible  to modify the height of the overlay?

### ISSUE THREAD

**Issue by gipsh** [sockeqwe/debugoverlay#4]
Currently is 2 lines, it would be great if i can configure the heigh of the overlay. 

Thanks!

**Comment by sockeqwe** [comment]
Yes, this project haven't got the love it deserves. We should add some primitive styling and customization with xml attributes. Each app could then override those attributes.

Wanna send a pull request?
